selftests/functional/test_basic.py

```diff
@@ -57,7 +57,8 @@ def test(self):
 class FakeStatusTest(Test):
     def run_avocado(self):
         super(FakeStatusTest, self).run_avocado()
-        self.status = 'not supported'
+        # Please do NOT use this, ever...
+        self._Test__status = 'not supported'
```

Comment:
You can explain that this is only for unittest checking purposes :) Just a suggestion though.